{"trial_id": "3", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "units_1", "default": null, "conditions": [], "min_value": 64, "max_value": 256, "step": 64, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "dropout_1", "default": 0.2, "conditions": [], "min_value": 0.2, "max_value": 0.5, "step": 0.1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_2", "default": null, "conditions": [], "min_value": 32, "max_value": 128, "step": 32, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "dropout_2", "default": 0.2, "conditions": [], "min_value": 0.2, "max_value": 0.5, "step": 0.1, "sampling": "linear"}}, {"class_name": "Choice", "config": {"name": "learning_rate", "default": 0.01, "conditions": [], "values": [0.01, 0.001, 0.0001], "ordered": true}}], "values": {"units_1": 64, "dropout_1": 0.2, "units_2": 32, "dropout_2": 0.2, "learning_rate": 0.01}}, "metrics": {"metrics": {"loss": {"direction": "min", "observations": [{"value": [0.8602863550186157], "step": 1}]}, "mae": {"direction": "min", "observations": [{"value": [0.7042406797409058], "step": 1}]}, "val_loss": {"direction": "min", "observations": [{"value": [0.8245664238929749], "step": 1}]}, "val_mae": {"direction": "min", "observations": [{"value": [0.6709668636322021], "step": 1}]}}}, "score": 0.8245664238929749, "best_step": 1, "status": "COMPLETED", "message": "Traceback (most recent call last):\n  File \"C:\\Users\\user\\anaconda3\\lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"C:\\Users\\user\\anaconda3\\lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 239, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n  File \"C:\\Users\\user\\anaconda3\\lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py\", line 314, in run_trial\n    obj_value = self._build_and_fit_model(trial, *args, **copied_kwargs)\n  File \"C:\\Users\\user\\anaconda3\\lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py\", line 232, in _build_and_fit_model\n    model = self._try_build(hp)\n  File \"C:\\Users\\user\\anaconda3\\lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py\", line 164, in _try_build\n    model = self._build_hypermodel(hp)\n  File \"C:\\Users\\user\\anaconda3\\lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py\", line 155, in _build_hypermodel\n    model = self.hypermodel.build(hp)\n  File \"C:\\Users\\user\\AppData\\Local\\Temp\\ipykernel_22348\\2919771365.py\", line 2, in <lambda>\n    lambda hp: build_model(hp, input_shape),\n  File \"C:\\Users\\user\\AppData\\Local\\Temp\\ipykernel_22348\\2016781147.py\", line 2, in build_model\n    model = keras.Sequential([\nNameError: name 'keras' is not defined\n"}